docs/guides/insar_product_guide.md

```diff
@@ -52,7 +52,10 @@ There is an option to apply a **water mask**. This mask includes coastal waters
 
 A GeoTIFF of the water mask is always included with the InSAR product package, but when this option is selected, the conditional water mask will be applied along with coherence and intensity thresholds during the phase unwrapping process. Water masking is turned off by default. 
 
-The water mask is generated by the HyP3 team using data from [OpenStreetMaps](https://www.openstreetmap.org/about){target=_blank} and/or [ESA WorldCover](https://esa-worldcover.org/en/about/about){target=_blank} depending on location. Areas within Canada, Alaska, and Russia are primarily covered by ESA WorldCover data, while the rest of the world is covered by OpenStreetMaps data. Water masks were previously generated from the GSHHG dataset.
+The water mask is generated by the HyP3 team using data from [OpenStreetMaps](https://www.openstreetmap.org/about){target=_blank} and/or [ESA WorldCover](https://esa-worldcover.org/en/about/about){target=_blank} depending on location. Areas within Canada, Alaska, and Russia are primarily covered by ESA WorldCover data, while the rest of the world is covered by OpenStreetMaps data. 
```

Comment:
```suggestion
The water mask is generated by the HyP3 team using data from [OpenStreetMaps](https://www.openstreetmap.org/about){target=_blank} and/or [ESA WorldCover](https://esa-worldcover.org/en/about/about){target=_blank} depending on location. Areas within Canada, Alaska, and Russia are primarily covered by ESA WorldCover data, while the rest of the world is covered by OpenStreetMaps data.

The water mask is available for all longitudes, and from -85 to 85 degrees latitude. Thus, the water mask is not available for the centers of the North Pole and South Pole.
```

Reply:
Addressed in [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/423/commits/69eb243b1b4c2ca630adf8af3bfa13ec2ba30ab6)